scripts/gap9-run.sh

```
@@ -0,0 +1,541 @@
+
+# USB/IP device settings
+USBIP_HOST="host.docker.internal"
+USBIP_VENDOR="15ba"
+USBIP_PRODUCT="002b"
+
+# SDK and cache directories
+WORK_DIR="."
+CACHE_FOLDER=".cache"
+
+# SSH key gap9 container
+SSH_PRIVATE_KEY="~/.ssh/id_ed25519"
```

Shellcheck (0.11.0)

[warning] Tilde does not expand in quotes. Use $HOME.

(SC2088)